examples/fiber/index.html

```diff
@@ -19,26 +19,155 @@
     </div>
     <script src="../../build/react.js"></script>
     <script src="../../build/react-dom-fiber.js"></script>
-    <script>
-      function ExampleApplication(props) {
-        var elapsed = Math.round(props.elapsed  / 100);
-        var seconds = elapsed / 10 + (elapsed % 10 ? '' : '.0' );
-        var message =
-          'React has been successfully running for ' + seconds + ' seconds.';
-
-        return React.DOM.p(null, message);
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.24/browser.min.js"></script>
+    <script type="text/babel">
+      var dotStyle = {
+        position: 'absolute',
+        background: '#61dafb',
+        font: 'normal 15px sans-serif',
+        textAlign: 'center',
+        cursor: 'pointer',
+      };
+
+      var containerStyle = {
+        position: 'absolute',
+        transformOrigin: '0 0',
+        left: '50%',
+        top: '50%',
+        width: '10px',
+        height: '10px',
+        background: '#eee',
+      };
+
+      var targetSize = 25;
+
+      class Dot extends React.Component {
+        constructor() {
+          super();
+          this.state = { hover: false };
+        }
+        enter() {
+          this.setState({
+            hover: true
+          });
+        }
+        leave() {
+          this.setState({
+            hover: false
+          });
+        }
+        render() {
+          var props = this.props;
+          var s = props.size * 1.3;
+          var idx = props.idx;
+          var style = {
+            ...dotStyle,
+            width: s + 'px',
+            height: s + 'px',
+            left: (props.x) + 'px',
+            top: (props.y) + 'px',
+            borderRadius: (s / 2) + 'px',
+            lineHeight: (s) + 'px',
+            background: this.state.hover ? '#ff0' : dotStyle.background
+          };
+          return (
+            <div style={style} onMouseEnter={() => this.enter()} onMouseLeave={() => this.leave()}>
+              {this.state.hover ? 'X' : props.text}
```

Comment:
It would actually be nice to use to use something from props here since that shows that we're able to reuse previous props for a new condition. Maybe `'"' + props.text + '"'` or something?